src/main/java/PersonalBudget/common/util/Date.java

```diff
@@ -0,0 +1,36 @@
+package PersonalBudget.common.util;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+
+import static java.time.temporal.TemporalAdjusters.firstDayOfYear;
+import static java.time.temporal.TemporalAdjusters.lastDayOfYear;
+
+public class Date {
```

Comment:
Date is a java.util class. I strongly suggest not to repeat the names of common java classes. You could call it PersonalBudgetDateUtils to make a distinction between your class and the standard ones.